### ISSUE TITLE
Audio Pulsing/Dropouts in bad network connection situations

### ISSUE THREAD

**Issue by KlienVo** [wildyeast/sadiss#83]
In Athens during the workshop/showing with approx. 45 clients connected and somewhat shaky netework connectivity, the following two tracks had serious 1s pulsing or complete audio dropouts:
ParkChoir_EN_GR_2023-06-09
LenasDoor_EN-GR_2023-06-09

For some telephones it worked without problems, for other there was pulsing, while otheres again did not get ANY partials at all. -> hints at a client/connection issue (but - of course - I may be wrong).

In another, smaller workshop in Athen we had similar dropout problems with only approx. 15 people connected.

both tracks had (in better network conditions) performed decently previously.

**Comment by viruslight** [comment]
I am currently having a really hard time to reproduce this. Throttling the network to slow speeds (downloading at 50 Kbps) does not have any effects, it seems. I tried with the tracks mentioned above, and the 2x stretched one from #82. The throttling is only simulated though. In a real life setting the connection could cut out completely for short amounts of time, for example.
I throttled the network for both the admin_client and the app (running in the browser) via a setting in the Firefox dev tools (see https://firefox-source-docs.mozilla.org/devtools-user/network_monitor/throttling/index.html).
It is somewhat conceivable that the fixes for #81 and #82 do help with this as well, but it's impossible to tell without a reliable way of reproducing this.
